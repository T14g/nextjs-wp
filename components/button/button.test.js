import React from "react";
import renderer from 'react-test-renderer';
import Button from './button.component';

it('Render Button correctly', () => {
    const tree = renderer.create(<Button text="Button Text" />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
