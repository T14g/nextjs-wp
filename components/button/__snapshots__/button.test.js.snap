// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Button correctly 1`] = `
<button
  className="buttonstyles__ButtonStyles-sc-9oe4d1-0 bwQuNq"
>
  Button Text
</button>
`;
